cachito/web/models.py

```diff
@@ -47,6 +47,33 @@ class Request(db.Model):
     def __repr__(self):
         return '<Request {0!r}>'.format(self.id)
 
+    @property
+    def state(self):
```

Comment:
Should `state` be `latest_state_name` and `latest_state` be `latest_state_object` ? Both of them return information about the latest state.